pkg/cli/subcommands/autoscalerlist/command.go

```diff
@@ -66,14 +68,24 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 			)
 		},
 	}
+	cmd.Flags().BoolVarP(&cliParams.localstore, "localstore", "v", false, "print autoscaling localstore debug info")
```

Comment:
if the user uses this flag, it will only print out the metrics store info and not the autoscaler list info - do we want it to print both? if not should we change the short flag to not be `-v` (that seems like we would provide more verbose output to the autoscaler list, not provide a different output)

not sure what the other commands in the agent are like, but wondering if it makes more sense to have this called from a separate command instead of a flag